std/haxe/format/JsonParser.hx

```diff
@@ -37,7 +37,7 @@ class JsonParser {
 		JSON objects are parsed into anonymous structures and JSON arrays
 		are parsed into `Array<Dynamic>`.
 
-		If given `str` is not valid JSON, an exception will be thrown.
+		If given `str` is not a valid JSON, an exception will be thrown.
```

Comment:
That one seems worse than original wording to me.

Reply:
it didn't seem right to me compared to now to be honest

Reply:
For example, `var foo = false;` is not "a valid Haxe", it's "valid Haxe".

Reply:
fair enough